src/views/v2/rsvp/ari/sidebar/guest-list/guest-template.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * This template renders the RSVP ARI sidebar guest list item JS template.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/sidebar/guest-list/item-template.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+<script
+	class="tribe-tickets__rsvp-ar-guest-list-item--template"
+	id="tmpl-tribe-tickets__rsvp-ar-guest-list-item--template-<?php echo esc_attr( $rsvp->ID ); ?>"
+	type="text/template"
+>
+	<li
+		class="tribe-tickets__rsvp-ar-guest-list-item"
+		data-guest-number="{{data.attendee_id + 1}}"
```

Comment:
do we actually use this in the JS? I also don't see this in the `v2/rsvp/ari/sidebar/guest-list/guest.php` on the `<li>` element.

Reply:
Oh right, I switched in favor of the button for this section.